css/style.css

```diff
@@ -1,9 +1,20 @@
-
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
-
+ :root {
```

Comment:
Setting the variables here is excellent work, makes changing things a lot easier!